stream_alert/alert_processor/outputs.py

```diff
@@ -0,0 +1,343 @@
+'''
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+
+import json
+import logging
+import os
+
+from abc import abstractmethod
+from collections import OrderedDict
+from datetime import datetime
+
+import boto3
+
+from stream_alert.alert_processor.output_base import StreamOutputBase, OutputProperty
+
+logging.basicConfig()
+LOGGER = logging.getLogger('StreamOutput')
+
+STREAM_OUTPUTS = {}
```

Comment:
I always like to document what's in a dictionary. What are the types of the keys and values?